extensions/who-deploys-most-often/app.R

```diff
@@ -130,6 +170,49 @@ server <- function(input, output, session) {
     ) |>
       formatDate(columns = "Time of Latest Deploy", method = "toLocaleString")
   })
+
+  output$total_active_content_plot <- renderPlotly(
+    ggplotly(
+      summarized_data() |>
+        arrange(desc(active_content)) |>
+        slice_head(n = 25) |>
+        ggplot() +
+        geom_col(aes(x = reorder(full_name, active_content), y = active_content)) +
```

Comment:
Super nit, but tooltips would be more readable if you use something like `text = paste("User: ", full_name, " Active Content: ", active_content)` across plots where `reorder()` is used. Currently the tooltip displays:
```
reorder(full_name, active_content): <full_name_value>
active_content: <active_content_value>
```

Reply:
:100: agree with @dethmasque on this